src/AlohaKit.Gallery/AlohaKit.Gallery.csproj

```diff
@@ -64,9 +57,18 @@
 	  <ProjectReference Include="..\AlohaKit\AlohaKit.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.61" />
+	  <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.61" />
+	  <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
+	  <PackageReference Include="Nerdbank.GitVersioning" Version="3.6.139" />
```

Comment:
these references should not be here they should only be in the Directory.Build.props